app/mailers/deploy_mailer.rb

```diff
@@ -17,7 +17,8 @@ def bypass_email(stage, deploy, user)
     to_email = [BuddyCheck.bypass_email_address]
     to_email << BuddyCheck.bypass_jira_email_address if BuddyCheck.bypass_jira_email_address
 
-    mail(to: to_email, cc: user.email, subject: subject)
+    cc_email = [user.email, BuddyCheck.bypass_retroactive_approval_email]
```

Comment:
We are the only people I know who use the Buddy functionality, so it's nice to keep it separate.